frontend/src/main/pages/CoursesShowPage.js

```diff
@@ -0,0 +1,57 @@
+
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import CoursesTable from 'main/components/Courses/CoursesTable';
+import { useBackend } from 'main/utils/useBackend';
+import { useCurrentUser } from 'main/utils/currentUser';
+
+export default function CoursesShowPage() {
+    let { id } = useParams();
+    const { data: currentUser } = useCurrentUser();
+
+    const { data: courses, error: _error, status: _status } =
+        useBackend(
+            [`/api/courses?id=${id}`],
+            {
+                method: "GET", url: "/api/courses/get",
+                params: {
+                    id
+                },
+            },
+    []
+        );
+        let checkLength = [];
+        if (courses && courses.length !== 0) {
+            checkLength = [courses];
+        } else if (courses) {
+            checkLength = courses;
+        }
+
+    return (
+        <BasicLayout>
+            <div className="pt-2">
+                <h1>Individual Course Information</h1>
+                <CoursesTable courses={checkLength} currentUser={currentUser} />
```

Comment:
```suggestion
                <CoursesTable courses={course ? [course] : []} currentUser={currentUser} />
```

This says: if you have one course, put it in an array which is what the CoursesTable expects.  If you don't have one course, then just display an empty list.